dask_planner/src/parser.rs

```diff
@@ -577,4 +620,33 @@ impl<'a> DaskParser<'a> {
             },
         })))
     }
+
+    /// Parse Dask-SQL ANALYZE TABLE <table>
+    fn parse_analyze_table(&mut self) -> Result<DaskStatement, ParserError> {
+        let table_factor = self.parser.parse_table_factor()?;
+        // parse_table_factor parses the following keyword as an alias, so we need to go back a token
+        // TODO: open an issue in sqlparser around this when possible
+        self.parser.prev_token();
```

Comment:
@jdye64 and I noticed that for a statement like `ANALYZE TABLE df COMPUTE STATISTICS FOR ALL COLUMNS`, sqlparser consumes and interprets `COMPUTE` as an alias for `df` even though there is no `AS` keyword. Not sure if this is expected behavior for `parse_table_factor`, so I added this comment in case we want to open up an issue around that later on.

cc @andygrove

Reply:
This does seem like a bug since `COMPUTE` is defined as a keyword in sqlparser-rs.

Reply:
Also, ANALYZE TABLE should really just be looking for an identifier for the table name and not call `parse_table_factor`.